mart/callbacks/adversary_connector.py

```diff
@@ -62,6 +62,7 @@ def __init__(
         val_adversary: Callable = None,
         test_adversary: Callable = None,
         batch_c15n: Callable = None,
+        module_step_fn: str = "training_step",
```

Comment:
This enables us to use anomalib's implementation of WinClip, which only implements a `validation_step` function.